docs/smart-contracts/getting-started/create-an-app.mdx

```diff
@@ -255,20 +255,21 @@ Now let's add a new component to the `src/components` directory called `ConnectF
     return publicKey;
   }
 
-  async function setLoggedIn() {
-    const publicKey = await getPk();
+  async function setLoggedIn(publicKey: string) {
     ellipsis.innerHTML = `Signed in as ${publicKey}`;
     ellipsis.title = publicKey;
   }
 
   if (await isAllowed()) {
-    if (await getPk()) setLoggedIn();
+    const publicKey = await getPk();
+    if (publicKey) setLoggedIn(publicKey);
```

Comment:
Why this change?

Reply:
Change was to avoid the two await getUserInfo calls but in retrospect its cost is negligible so doesn't matter.